certify-service/src/test/java/io/mosip/certify/controller/VCIssuanceControllerTest.java

```diff
@@ -0,0 +1,195 @@
+        credentialProof.setProof_type("jwt");
+        credentialRequest.setProof(credentialProof);
+        CredentialDefinition credentialDefinition = new CredentialDefinition();
+        credentialDefinition.setType(Arrays.asList("VerifiableCredential", "SampleVerifiableCredential_ldp"));
+        credentialRequest.setCredential_definition(credentialDefinition);
+
+        mockMvc.perform(post("/vci/credential")
+                        .content(objectMapper.writeValueAsBytes(credentialRequest))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isBadRequest())
+                .andExpect(jsonPath("$.error").value(ErrorConstants.INVALID_VC_FORMAT));
+
+        credentialRequest.setFormat("  ");
+        mockMvc.perform(post("/vci/credential")
+                        .content(objectMapper.writeValueAsBytes(credentialRequest))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isBadRequest())
+                .andExpect(jsonPath("$.error").value(ErrorConstants.INVALID_VC_FORMAT));
+    }
+
+    @Test
+    public void getVC_withInvalidProof_thenFail() throws Exception {
+        CredentialRequest credentialRequest = new CredentialRequest();
+        credentialRequest.setFormat("jwt_vc_json");
+        CredentialDefinition credentialDefinition = new CredentialDefinition();
+        credentialDefinition.setType(Arrays.asList("VerifiableCredential", "SampleVerifiableCredential_ldp"));
+        credentialRequest.setCredential_definition(credentialDefinition);
+
+        credentialRequest.setProof(null);
+        mockMvc.perform(post("/vci/credential")
+                        .content(objectMapper.writeValueAsBytes(credentialRequest))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isBadRequest())
+                .andExpect(jsonPath("$.error").value(ErrorConstants.INVALID_PROOF));
+
+        CredentialProof credentialProof = new CredentialProof();
+        credentialRequest.setProof(credentialProof);
+        mockMvc.perform(post("/vci/credential")
+                        .content(objectMapper.writeValueAsBytes(credentialRequest))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isBadRequest())
+                .andExpect(jsonPath("$.error").value(ErrorConstants.UNSUPPORTED_PROOF_TYPE));
+
+
+        credentialProof.setProof_type("  ");
+        credentialRequest.setProof(credentialProof);
+        mockMvc.perform(post("/vci/credential")
+                        .content(objectMapper.writeValueAsBytes(credentialRequest))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isBadRequest())
+                .andExpect(jsonPath("$.error").value(ErrorConstants.UNSUPPORTED_PROOF_TYPE));
+    }
+
+    @Test
+    public void test_getVC_withInvalidNonceException_thenFail() throws Exception {
```

Comment:
Let us follow consistent test method naming convention. 
methodToTest_testCaseName_testResult
Eg: getVerifiableCredential_withValidProof_thenPass()

Reply:
changed the method names to this format